crates/plex-api/tests/server.rs

```diff
@@ -925,6 +931,207 @@ mod online {
         assert!(matches!(err, Error::ItemNotFound));
     }
 
+    #[plex_api_test_helper::online_test]
+    async fn download(#[future] server: Server) {
+        // The first 32 bytes of white_noise_720p.mkv
+        const MKV_START: [u8; 32] = [
+            0x1A, 0x45, 0xDF, 0xA3, 0xA3, 0x42, 0x86, 0x81, 0x01, 0x42, 0xF7, 0x81, 0x01, 0x42,
+            0xF2, 0x81, 0x04, 0x42, 0xF3, 0x81, 0x08, 0x42, 0x82, 0x88, 0x6D, 0x61, 0x74, 0x72,
+            0x6F, 0x73, 0x6B, 0x61,
+        ];
+        // The last 16 bytes of white_noise_720p.mkv
+        const MKV_END: [u8; 16] = [
+            0xB3, 0x81, 0x2D, 0xB7, 0x8B, 0xF7, 0x81, 0x01, 0xF1, 0x82, 0x02, 0xE1, 0xF0, 0x82,
+            0x01, 0xA0,
+        ];
+        // The length of white_noise_720p.mkv
+        const MKV_LEN: u64 = 13400382;
+
+        // The first 16 bytes of white_noise_720p_h265.mp4
+        const MP4_START: [u8; 16] = [
+            0x00, 0x00, 0x00, 0x1C, 0x66, 0x74, 0x79, 0x70, 0x69, 0x73, 0x6F, 0x6D, 0x00, 0x00,
+            0x02, 0x00,
+        ];
+        // The length of white_noise_720p_h265.mkv
```

Comment:
A tiny copy&paste error :)

```suggestion
        // The length of white_noise_720p_h265.mp4
```